src/utils/date-utils.js

```diff
@@ -35,7 +35,7 @@ export function convertPotentialTimestampToSeconds(timestamp) {
   let timestampInSeconds;
   if (timestamp >= 1e14 || timestamp <= -1e14) {
     // Microseconds
-    timestampInSeconds = Math.floor(timestamp / 100000);
+    timestampInSeconds = Math.floor(timestamp / 1000000);
```

Comment:
Consider writing the literal as 1e6 or 1_000_000 for clarity (either style would have made the bug evident).